adminapi/request.py

```diff
@@ -91,7 +91,14 @@ def calc_signature(private_key, timestamp, data=None):
     Returns the signature as base64 encoded unicode, ready for transport.
     """
     message = calc_message(timestamp, data)
-    sig = private_key.sign_ssh_data(message.encode())
+    try:
+        sig = private_key.sign_ssh_data(message.encode())
+    except SSHException as e:
+        # There is a possbility, that the key cannot be used for authentication
+        # e.g., RSA keys for PIV Attestation
```

Comment:
In the PR conversation you mention that you fail to sign the request with the ECDSA key but in this comment you are mentioning the RSA key. With which key does it fail - maybe we can trace the root cause ?